package.json

```diff
@@ -232,13 +232,14 @@
     "@perf-profiler/reporter": "^0.9.0",
     "@perf-profiler/types": "^0.8.0",
     "@pmmmwh/react-refresh-webpack-plugin": "^0.5.15",
-    "@react-native-community/cli": "15.0.1",
-    "@react-native-community/cli-platform-android": "15.0.1",
-    "@react-native-community/cli-platform-ios": "15.0.1",
     "@react-native-community/eslint-config": "3.2.0",
     "@react-native/babel-preset": "0.76.3",
     "@react-native/metro-config": "0.76.3",
     "@react-navigation/devtools": "^6.0.10",
+    "@rnef/cli": "0.4.1",
+    "@rnef/plugin-metro": "0.4.1",
```

Comment:
how does this interact with [this other proposal](https://github.com/Expensify/App/issues/58311) to use re.pack instead of metro?